server/database/db.js

```diff
@@ -7,7 +7,20 @@ const connection = mysql.createConnection(config);
 connection.connect((err) => {
   if (err) {
     console.log(err);
+  } else {
+    console.log('database connected!');
   }
 });
 
-module.exports.connection = connection;
+const deleteReview = (id, callback) => {
+  const queryStr = `DELETE FROM reviews WHERE id=${id}`;
+  connection.query(queryStr, (err, result) => {
+    if (err) { callback(err); }
+    else if (result) { callback(null, result); }
```

Comment:
Do you need the if statement for results? Won't you have that if the error conditional fails?